src/bin/pg_autoctl/cli_common.c

```diff
@@ -494,12 +494,6 @@ cli_create_node_getopts(int argc, char **argv,
 		exit(EXIT_CODE_BAD_ARGS);
 	}
 
-	if (errors > 0)
-	{
-		commandline_help(stderr);
-		exit(EXIT_CODE_BAD_ARGS);
-	}
-
```

Comment:
That comes as a surprise, at first re-read of the code I would say we still need this...

Reply:
This exact check is already done in line 398 and after that errors is not incremented in this function anymore.